### DISCUSSION TITLE
Flux Gap

### DISCUSSION THREAD

**Discussion by MohammadReza0GH** [phoebe-project/phoebe2#867] [Questions]
![hhhhhhhhhhhhhhhhhhhhhhhhhhh](https://github.com/phoebe-project/phoebe2/assets/167889059/5affdc8b-70ff-4592-ba40-4f101aff88c1)

why there is gap in data and fitted lc?

**Comment by aprsa** [comment]
http://phoebe-project.org/docs/latest/tutorials/pblum

**Reply by MohammadReza0GH** [reply]
tnx.
